apps/src/args.rs

```diff
@@ -68,6 +70,8 @@ pub struct CommonArgs {
 /// --dgram-proto PROTO         DATAGRAM application protocol.
 /// --dgram-count COUNT         Number of DATAGRAMs to send.
 /// --dgram-data DATA           DATAGRAM data to send.
+/// --max-active-cids NUM       Maximum number of active Connection IDs.
+/// --migrate                   Enable active connection migration.
```

Comment:
I'm not a huge fan of this flag name since it can be overloaded. How about `--enable-active-migration`?

Reply:
Naming is complicated :) but yes, `--enable-active-migration` actually better describes what this option does. Will update that and the following occurrences.